From 8159b7e887b2ff40a279ac89ba7a6e04b6f7054e Mon Sep 17 00:00:00 2001
From: Kalyan Alle <kalyan.alle@amd.com>
Date: Tue, 24 Jul 2018 17:41:27 +0530
Subject: [PATCH 4095/4131] drm/amd/amdgpu: workaround to resolve the mgpu

not functional issue.

Revert ttm_sg_tt_init and roll back to ttm_dma_tt_init

Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
index 11ac7f2..71a5c67 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_ttm.c
@@ -1047,11 +1047,7 @@ static struct ttm_tt *amdgpu_ttm_tt_create(struct ttm_buffer_object *bo,
 		return NULL;
 	}
 	gtt->ttm.ttm.func = &amdgpu_backend_func;
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 17, 0) || !defined(BUILD_AS_DKMS)
-        if (ttm_sg_tt_init(&gtt->ttm, bo, page_flags)) {
-#else
         if (ttm_dma_tt_init(&gtt->ttm, bo, page_flags)) {
-#endif
 		kfree(gtt);
 		return NULL;
 	}
-- 
2.7.4

